package.json

```diff
@@ -8,22 +8,15 @@
     "@testing-library/react": "10.3.0",
     "@testing-library/user-event": "12.0.6",
     "axios": "^0.19.2",
-    "babel-plugin-dynamic-import-node": "^2.0.0",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "express": "^4.17.1",
-    "file-loader": "^2.0.0",
-    "ignore-styles": "^5.0.1",
     "node-sass": "^4.14.1",
     "react": "16.13.1",
     "react-app-rewired": "^2.1.6",
     "react-dom": "16.13.1",
-    "react-loadable": "^5.3.1",
     "react-redux": "^7.2.0",
     "react-router-dom": "5.2.0",
-    "react-scripts": "3.4.1",
+    "react-scripts": "^3.4.3",
```

Comment:
These dont belong in your PR - im guessing you merged main branch

Reply:
Not too sure, we might have to go over rebasing again. Theres something that's going on with the commit histories for most of us.